src/scripts/import_assessment.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# import_assessment.sh ASSESSMENT_FILE
+
+# This script simplifies the process of importing an assessment JSON file
+# into the GoPhish server running in the local Docker composition.
+
+set -o nounset
+set -o pipefail
+
+if [ $# -ne 1 ] || [ "$1" == "-h" ] || [ "$1" == "--help" ]
+then
+    echo "Usage: import_assessment.sh ASSESSMENT_FILE"
+    exit 255
+fi
+
+ASSESSMENT_FILE=$1
+ASSESSMENT_FILE_BASE=$(basename "$ASSESSMENT_FILE")
+ASSESSMENT_FILE_DIR=$(readlink -f "$ASSESSMENT_FILE" | xargs dirname)
+
+GOPHISH_COMPOSITION="/var/pca/pca-gophish-composition/docker-compose.yml"
+GOPHISH_URL="https://gophish:3333"
+COMPLETE_CAMPAIGN_SCRIPT="/var/pca/pca-gophish-composition/src/scripts/complete_campaign.sh"
+
+# Fetch GoPhish API key
+API_KEY=$(docker-compose -f "$GOPHISH_COMPOSITION" exec -T gophish get-api-key)
+api_key_rc="$?"
+if [ "$api_key_rc" -ne 0 ]
+then
+  echo "ERROR: Failed to obtain GoPhish API key from Docker composition."
+  echo "Exiting without importing."
+  exit 1
+fi
+
+# Run gophish-import in the Docker composition
+docker-compose -f "$GOPHISH_COMPOSITION" run --rm \
+  --volume "$ASSESSMENT_FILE_DIR":/home/cisa gophish-tools \
+  gophish-import "$ASSESSMENT_FILE_BASE" "$GOPHISH_URL" "$API_KEY"
+import_rc="$?"
+if [ "$import_rc" -eq 0 ]
+then
+  echo "Assessment successfully imported from $ASSESSMENT_FILE!"
+  echo ""
+else
+  echo "ERROR: Assessment import from $ASSESSMENT_FILE failed!"
+  exit $import_rc
+fi
+
+# TODO - REMOVE THIS TEMP HACK
+ASSESSMENT_FILE="$1.json"
```

Comment:
This is pretty ugly.  Why not use two variables?  Call the other variable `ASSESSMENT_ID` or something similar and create `ASSESSMENT_FILE="${ASSESSMENT_ID}.json"` from it.  (Use whatever names make sense.)

Reply:
This was just a temporary thang until @bjb28 updated one of the scripts in [`gophish-tools`](https://github.com/cisagov/gophish-tools), which he did [here](https://github.com/cisagov/gophish-tools/commit/5e2dc21614d2e6a7a8d313b510ad316a18de6a04#diff-732e89c3e3a7866d8c19dd0bfe0d38a9L276).  I just forgot to undo my hack before I put in this PR.  That has been rectified with https://github.com/cisagov/pca-gophish-composition-packer/pull/13/commits/b799060c2a764a7fd5f231bab4d2e5172e08fc8a.